src/backend/looped/self/super_jump.cpp

```diff
@@ -0,0 +1,18 @@
+#include "natives.hpp"
+#include "backend/looped_command.hpp"
+#include "util/entity.hpp"
+
+namespace big
+{
+	class super_jump_looped : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			MISC::SET_SUPER_JUMP_THIS_FRAME(self::ped);
```

Comment:
I remember that this would cause metrics from being sent out.

Reply:
ill use the task_jump native then, that should work